README.md

```diff
@@ -86,6 +86,15 @@ edgelabel = 1:Graphs.ne(g)
 gplot(g, edgelabel=edgelabel, nodelabel=nodelabel, edgelabeldistx=0.5, edgelabeldisty=0.5)
 ```
 
+## Add dash style to edges
+```julia
+using Measures
+gplot(g, edgedashstyle=[5mm, 2mm])
+# vary edge styles
+using Compose
+gplot(g, EDGELINEWIDTH=0.7, edgedashstyle=[[10px, 5px], [10px, 5px, 3px, 5px], [4px, 4px], []])
```

Comment:
The use of this kwarg should be properly documented. When vectors are passed for each edge, the values define the sequence of dashes and spaces. For example, the first edge will have 10px solid dash with 5px spacing in between; the second edge will have the following alternating sequence: 10px solid - 5 px space - 3px solid - 5 px space - etc...